bin/hsw-cli

```diff
@@ -586,99 +586,87 @@ class CLI {
+        this.log('  $ address [account-name]: Derive new address.');
         this.log('  $ balance: Get wallet balance.');
+        this.log('  $ block [height]: View wallet block.');
+        this.log('  $ blocks: List wallet blocks.');
+        this.log('  $ change [account-name]: Derive new change address.');
+        this.log('  $ coins: View wallet coins.');
+        this.log('  $ dump [address]: Get wallet key WIF by address.');
+        this.log('  $ get: View wallet.');
         this.log('  $ history: View TX history.');
+        this.log('  $ import [wif|hex]: Import private or public key.');
+        this.log('  $ key [address]: Get wallet key by address.');
+        this.log('  $ listen: Listen for events.');
+        this.log('  $ lock: Lock wallet.');
+        this.log('  $ mktx [address] [value]: Create transaction.');
         this.log('  $ pending: View pending TXs.');
-        this.log('  $ coins: View wallet coins.');
-        this.log('  $ account list: List account names.');
-        this.log('  $ account create [account-name]: Create account.');
-        this.log('  $ account get [account-name]: Get account details.');
-        this.log('  $ address: Derive new address.');
-        this.log('  $ change: Derive new change address.');
         this.log('  $ retoken: Create new api key.');
         this.log('  $ send [address] [value]: Send transaction.');
-        this.log('  $ mktx [address] [value]: Create transaction.');
+        this.log('  $ shared add [xpubkey]: Add key to wallet.');
+        this.log('  $ resendwallet [id]: Resend pending transactions for a single wallet.');
+        this.log('  $ shared remove [xpubkey]: Remove key from wallet.');
         this.log('  $ sign [tx-hex]: Sign transaction.');
-        this.log('  $ zap [age?]: Zap pending wallet TXs.');
-        this.log('  $ abandon [hash]: Abandon a transaction.');
         this.log('  $ tx [hash]: View transaction details.');
-        this.log('  $ blocks: List wallet blocks.');
-        this.log('  $ block [height]: View wallet block.');
+        this.log('  $ unlock [passphrase] [timeout?]: Unlock wallet.');
         this.log('  $ view [tx-hex]: Parse and view transaction.');
-        this.log('  $ import [wif|hex]: Import private or public key.');
         this.log('  $ watch [address]: Import an address.');
-        this.log('  $ key [address]: Get wallet key by address.');
-        this.log('  $ dump [address]: Get wallet key WIF by address.');
-        this.log('  $ lock: Lock wallet.');
-        this.log('  $ unlock [passphrase] [timeout?]: Unlock wallet.');
-        this.log('  $ resend: Resend pending transactions.');
+        this.log('  $ zap [age]: Zap pending wallet TXs.');
+        this.log('');
+        this.log('If node is run with wallet-auth flag, wallet commands require authorization token.');
+        this.log('Admin commands require admin permissions for provided authorization token:');
```

Comment:
I am not sure if this sentence means what it should?

You don't have per-token admin permissions in the wallet, you just have a single admin token that you need to use for any `admin` related functionality.

Reply:
https://hsd-dev.org/api-docs/#wallet-authentication
There are wallet-specific tokens which you need to provide, if wallet-auth is true.

Reply:
I just meant by this lines that if wallet-auth is true, then you must provide token no matter what, if it happens to be 'non-admin' token, then you will not be able to call 'admin' commands.